<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qdc</class>
 <widget class="QWidget" name="qdc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_clearChannel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Channel Buffer</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="qdc_profile" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>651</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>nBins: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>575</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Min.:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>615</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Max.:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_qdc_nbins">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>525</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4096</number>
   </property>
   <property name="value">
    <number>4096</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of events:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_qdc_nevents">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>365</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="value">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_qdc_startprofile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stopprofile">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_qdc_vmin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>4096.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_qdc_vmax">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>610</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>4096.000000000000000</double>
   </property>
   <property name="value">
    <double>4096.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clearProfile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Histogram</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_qdc_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotary Switches</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qdc_module_type">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>75</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_qdc_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Module Type</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_qdc_rotary_switches">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>45</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>13056</number>
   </property>
   <property name="displayIntegerBase">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_qdc_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Board Panel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_qdc_display" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="radioButton_qdc_enable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>2</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>enable / disable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel: </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_qdc_channelN">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>40</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>* 0.1 (pC)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set it</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_qdc_threshold">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>35</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4080</number>
    </property>
    <property name="singleStep">
     <number>16</number>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qdc_file_name">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>665</y>
     <width>521</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_qdc_file_name">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>665</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_qcd_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>670</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_qdc_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Channel Configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_qdc_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Run Configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_qdc_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Histograme</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input channel:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_input_channel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>335</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_plot">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>30</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Plot</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
